assets/src/edit-story/components/panels/utils/getCommonValue.js

```diff
@@ -14,26 +14,35 @@
  * limitations under the License.
  */
 
+/**
+ * Internal dependencies
+ */
+import { MULTIPLE_VALUE } from '../../form';
+
 /**
  * Get the common value `property` for all objects in `list`, if they
- * in fact are all the same. If they are not all equal, return an empty string.
- *
- * Example usage:
- * ```
- * getCommonValue( [ { a: 1 }, { a:  1  } ], 'a' );  // returns: 1
- * getCommonValue( [ { a: 1 }, {        } ], 'a' );  // returns: ''
- * getCommonValue( [ { a: 1 }, { a:  2  } ], 'a' );  // returns: ''
- * getCommonValue( [ { a: 1 }, { a: '1' } ], 'a' );  // returns: ''
- * ```
+in fact are all the same. If they are not all equal, return an empty string.
+ 
+Example usage:
+```
+getCommonValue( [ { a: 1 }, { a:  1  } ], 'a' );  // returns: 1
+getCommonValue( [ { a: 1 }, {        } ], 'a' );  // returns: MULTIPLE_VALUE
+getCommonValue( [ { a: 1 }, { a:  2  } ], 'a' );  // returns: MULTIPLE_VALUE
+getCommonValue( [ { a: 1 }, { a: '1' } ], 'a' );  // returns: MULTIPLE_VALUE
+```
  *
- * @param {Array.<Object>} list  List of objects
- * @param {string} property Property to check on all objects
+ * @param {Array<Object>} list List of objects
+ * @param {string|function(Object):any} propertyOrFunc Property to check on all objects.
  * @return {any} Returns common value or empty string if not similar
  */
-function getCommonValue(list, property) {
-  const first = list[0][property];
-  const allMatch = list.every((el) => el[property] === first);
-  return allMatch ? first : '';
+function getCommonValue(list, propertyOrFunc) {
+  const func =
```

Comment:
Curious what's this change for -- allowing passing a function instead of a property. Doesn't look like it's currently being used in the PR, is I can see correctly.

Reply:
I removed the use case from here, but it was used like this:

```
const opacity = getCommonValue(selectedElements, ({opacity}) => typeof opacity === 'number' ? opacity : 1);
```

I thought it was more generic then adding more and more args to the `getCommonValue`, e.g. for default values, etc. But then I found the migration routine that ensured that it set opacity values everywhere, and removed the defaulting. That being said, this will likely be useful for other migrations. So I'd rather wait until I redo the other panels to confirm.